toolbox/load_agol_layer/src/utils.js

```diff
@@ -139,6 +153,22 @@ const getEsriJson = async (url) => {
   return genericJSONFetch({ url, method: "GET" });
 };
 
+const getTruncateMutation = (tableName) => `
```

Comment:
oh nice! i was wondering how the table names were going to come into play when i saw the updated config. 🚀